app/Http/Controllers/Dormitory/Printing/PrintAccountController.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace App\Http\Controllers\Dormitory\Printing;
+
+use App\Http\Controllers\Controller;
+use App\Mail\ChangedPrintBalance;
+use App\Models\Checkout;
+use App\Models\PaymentType;
+use App\Models\Semester;
+use App\Models\Transaction;
+use App\Models\User;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Contracts\Container\BindingResolutionException;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Mail;
+use Psr\Container\NotFoundExceptionInterface;
+use Psr\Container\ContainerExceptionInterface;
+
+class PrintAccountController extends Controller
+{
+    /**
+     * Updates balance of a `PrintAccount`.
+     * This can be done in two ways: either by topping up the account by giving money to an admin
+     * or by transfering money from one account to the other.
+     */
+    public function update(Request $request)
+    {
+        $request->validate([
+            'amount' => 'required|integer',
+            'user' => 'required|exists:users,id', // Normally this would be a path parameter for the PrintAccount, but we can't do that because of the limitations of blade templates
```

Comment:
I think the main point is that it would be a path param, not a request param. But still if you can tell me how I can do that in the blade, that would be nice.

Reply:
Ah, I see. I have an example how we handle that in the tabulator, with javascript: 
```
.ajax({
                    type: "POST",
                    url: "{{ route('internet.internet_accesses.extend', [':id']) }}".replace(':id', id),
                    ...})
```
You need some js to override the default form sending logic and replace the URL, but I think it's worth it here